go/libkb/wot.go

```diff
@@ -200,49 +220,30 @@ func fetchWot(mctx MetaContext, username *string) (res []serverWotVouch, err err
 	return response.Vouches, nil
 }
 
-func FetchMyWot(mctx MetaContext) (res []keybase1.WotVouch, err error) {
-	defer mctx.Trace("FetchMyWot", func() error { return err })()
-	serverVouches, err := fetchWot(mctx, nil)
+// FetchWotVouches gets vouches written for vouchee (if specified) by voucher (if
+// specified).
+func FetchWotVouches(mctx MetaContext, arg FetchWotVouchesArg) (res []keybase1.WotVouch, err error) {
+	vouches, err := fetchWot(mctx, arg.Vouchee, arg.Voucher)
 	if err != nil {
-		mctx.Debug("error fetching pending web-of-trust vouches: %s", err.Error())
+		mctx.Debug("error fetching web-of-trust vouches for %s: %s", arg.Vouchee, err.Error())
```

Comment:
errrrr. this isn't going to work. it's going to print either nil if you specified a voucher or the address of the memory if you specified a vouchee. neither is useful. i'd probably add a `String()` method to the arg object which just serializes something like `"voucher: nil, vouchee: merry"` and then
```suggestion
		mctx.Debug("error fetching web-of-trust vouches for %s: %s", arg.String(), err.Error())
```

Reply:
:(